app/views/users/sign_up.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>Sign Up</h1>
+
+<% if @user %>
+  <ul>
+    <% @user.errors.each do |column, error_message| %>
+      <div>
```

Comment:
Another super nit-picky thing, since this is in a `<ul>`, the next child should be the `<li>` not a `<div>`, it's ok to put a `<div>` in the `<li>` if you need to. And it's ok to style an `<li>` however you want.